crux-lucene/README.adoc

```diff
@@ -0,0 +1,32 @@
+= Crux Lucene
+
+Lucene module for Crux making use of https://lucene.apache.org/[Apache Lucene].
+
+`crux-lucene` runs in-process as part of the Crux node as a Crux module.
+
+.deps.edn
+[source,clojure]
+----
+juxt/crux-lucene {:mvn/version "20.09-1.11.0-alpha"}
+----
+
+.pom.xml
+[source,xml]
+----
+<dependency>
+    <groupId>juxt</groupId>
+    <artifactId>crux-lucene</artifactId>
+    <version>20.09-1.11.0-alpha</version>
+</dependency>
+----
+
+For more details, see the https://opencrux.com/reference/lucene.html[Crux Lucene documentation]
+
+== Scoring
+
+This module uses the default BM25Similarity similarity to calculate
```

Comment:
nit: I'm reliably informed the idiom for adoc is one sentence per line rather than wrapping at 80 :)

Reply:
hmm - I park that one for a wider adoc discussion! cc @malcolmsparks